cypress/e2e/checkout_spec.cy.js

```diff
@@ -101,6 +52,28 @@ describe('checkout', () => {
         .should('have.value', 'Shmoe')
         .get('textarea[name="checkout__form__comments"]').type('YOOO')
         .should('have.value', 'YOOO')
+      cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/graphql', {
+        statusCode: 200,
+        fixture: 'success.json'
+      }).as('createOrderForm')
+        .get('.checkout__form__submit').click()
```

Comment:
Shouldn't we wait for `createOrderForm` after this click?

Reply:
added